kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,92 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-01T00:00:00Z
+---
+
+`DeadlineExceeded` is a [gRPC Context error](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines) thrown when requests are not completed on time.
```

Comment:
```suggestion
`DeadlineExceeded` is a [gRPC error](https://grpc.io/docs/what-is-grpc/core-concepts/#deadlines) thrown when requests are not completed on time.
```

"context" is a go-specific concept (and appears in most deadline exceeded errors because our services are written in go)

Reply:
@DJSanti This is the differentiation I was trying to make as well. I think we want to capture @lorensr 's point about this.

Reply:
Gotcha. Will correct